edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -48,6 +48,7 @@
 COMB_INVALIDINPUT_ERROR = "Invalid input: {} not found in 'combinations' property"
 COMB_UNKOWN_ERROR = "Could not find a Combination named '{}' in '{}'"
 ATTRIBUTE_MISSING_ERROR = "Missing required attribute. Must specify either 'branch' or 'commit' for each <Source>."
+INVALID_ATTRIBUTE_ERROR = "Can not specify branch or commit or tag along with patchSet"

```

Comment:
The name of this error string is a little two generic can we change it to INVALID_COMBO_DEFINITION_ERROR